src/molecules/Table.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const StyledTr = styled.tr`
+  background-color: #e8eaed;
+  text-align: left;
+
+  :nth-child(2n) {
```

Comment:
Wow, nice technique. I wonder why taught you that ;)

Reply:
Yes! Thank you =)